Corona-Warn-App/src/main/res/values/styles.xml

```diff
@@ -683,4 +683,13 @@
         <item name="android:background">?selectableItemBackground</item>
     </style>
 
+    <style name="statusBadge">
+        <item name="android:textSize">14sp</item>
+        <item name="android:paddingStart">6dp</item>
+        <item name="android:paddingEnd">6dp</item>
+        <item name="android:minHeight">26dp</item>
+        <item name="android:gravity">center</item>
+        <item name="android:textColor">#FFFFFF</item>
```

Comment:
here you can add the minWidth of 40dp

Reply:
@schauersbergern  can you pls have a look ?

Reply:
Which minWidth? In the design there is just a minHeight of 26dp
https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.25?node-id=55153%3A40951